examples/elasticsearch/Examples.ElasticSearch/Examples.ElasticSearch.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'd suggest to remove this example from this PR. We haven't formulated a plan for how this repo would maintain examples in the long term. (too many examples, one per project, is going to be very difficult to maintain.).
Until the plan is sorted out, I'd recommend to not add examples.

Reply:
Hi @cijothomas ,

I removed the sample project as suggested. Can you re-trigger the build please?